PhysicsTools/NanoAOD/python/jetsAK8_cff.py

```diff
@@ -96,34 +96,35 @@
         btagDDBvLV2 = Var("bDiscriminator('pfMassIndependentDeepDoubleBvLV2JetTags:probHbb')",float,doc="DeepDoubleX V2(mass-decorrelated) discriminator for H(Z)->bb vs QCD",precision=10),
         btagDDCvLV2 = Var("bDiscriminator('pfMassIndependentDeepDoubleCvLV2JetTags:probHcc')",float,doc="DeepDoubleX V2 (mass-decorrelated) discriminator for H(Z)->cc vs QCD",precision=10),
         btagDDCvBV2 = Var("bDiscriminator('pfMassIndependentDeepDoubleCvBV2JetTags:probHcc')",float,doc="DeepDoubleX V2 (mass-decorrelated) discriminator for H(Z)->cc vs H(Z)->bb",precision=10),
-        deepTag_TvsQCD = Var("bDiscriminator('pfDeepBoostedDiscriminatorsJetTags:TvsQCD')",float,doc="DeepBoostedJet tagger top vs QCD discriminator",precision=10),
```

Comment:
I don't think we agreed to remove the DeepBoostedJet taggers...?

Reply:
@abenecke can you please weigh in here? Let me know if I removed the wrong thing, or who else we should discuss with about this first.

Reply:
We wanted to remove the DeepAK8 training, which should be the DeepBoosted nodes. These are created with training from pre-UL and are therefore not up to date. But we can discuss to keep them if needed?

Reply:
BTV also confirmed that we can remove DeepBoostedJet, see conversation here: https://gitlab.cern.ch/cms-nanoAOD/xpog-coordination/-/issues/61#note_6503093